core/src/main/java/brooklyn/location/access/PortForwardManagerImpl.java

```diff
@@ -38,12 +25,25 @@
 import brooklyn.location.basic.AbstractLocation;
 import brooklyn.mementos.LocationMemento;
 import brooklyn.util.collections.MutableMap;
-
 import com.google.common.base.Objects.ToStringHelper;
+import com.google.common.base.Predicate;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Lists;
 import com.google.common.net.HostAndPort;
+import org.slf4j.Logger;
```

Comment:
Try not to let Intellij reorganise the imports. The vast majority of our code uses the Eclipse defaults for imports, so we should stick to that in classes that use it.

You can set the formatting in Intellij to use the Eclipse defaults, but I'm not sure exactly how. Check with @richardcloudsoft ? If you find out, can you please update https://brooklyn.incubator.apache.org/v/latest/dev/env/ide/index.html

But I won't let that block merging of this PR.